README.md

```diff
@@ -19,9 +19,9 @@ Usage documentation exists for each major version. Don't know what version you'r
 
 ## Installation
 
-Make sure you have a working Go environment.  Go version 1.11+ is supported. [See the install instructions for Go](http://golang.org/doc/install.html).
+Using this package requires a working Go environment. [See the install instructions for Go](http://golang.org/doc/install.html).
 
-Go Modules are strongly recommended when using this package. [See the go blog guide on using Go Modules](https://blog.golang.org/using-go-modules).
```

Comment:
Go modules are production ready now, I think it's safe to make this statement stronger